`tn_normalize_sentence` does not exist in Rust

Currently

```rust
use text_processing_rs::tn_normalize_sentence;

fn main() {
    let normalized_text = tn_normalize_sentence("I have twenty one apples.");
    println!("{normalized_text}");
}
```

gives error

```sh
error[E0432]: unresolved import `text_processing_rs::tn_normalize_sentence`
 --> src/main.rs:6:5
  |
6 | use text_processing_rs::tn_normalize_sentence;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `tn_normalize_sentence` in the root
  |
help: a similar name exists in the module
  |
6 - use text_processing_rs::tn_normalize_sentence;
6 + use text_processing_rs::normalize_sentence;
  |
```

`normalize_sentence` exists, however `tn_normalize_sentence` seems not exist.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "text-processing-rs"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
 license = "Apache-2.0"
 description = "Inverse Text Normalization (ITN) — convert spoken-form ASR output to written form"

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@fluidinference/text-processing-rs",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "Inverse Text Normalization (ITN) — convert spoken-form ASR output to written form",
   "type": "module",
   "main": "pkg-web/text_processing_rs.js",

--- a/src/ffi.rs
+++ b/src/ffi.rs
@@ -168,7 +168,7 @@ pub extern "C" fn nemo_rule_count() -> u32 {
 /// Returns a static string, do not free.
 #[no_mangle]
 pub extern "C" fn nemo_version() -> *const c_char {
-    static VERSION: &[u8] = b"0.1.0\0";
+    static VERSION: &[u8] = concat!(env!("CARGO_PKG_VERSION"), "\0").as_bytes();
     VERSION.as_ptr() as *const c_char
 }
 

